packages/cosmic-swingset/src/anylogger-agd.js

```diff
@@ -0,0 +1,165 @@
+/* eslint-env node */
+import {
+  getEnvironmentOptionsList,
+  getEnvironmentOption,
+} from '@endo/env-options';
```

Comment:
Thinking about this specific kind of ambient authority as module parameterization is an interesting angle that @erights recently suggested, but... it's still dynamically scoped.

I suppose one can override it at Compartment boundaries rather than module boundaries. hm.

adding to the index to think about

  - #2160